static/js/pages/receipts.js

```diff
@@ -65,4 +68,25 @@ $(document).ready(function() {
       pagingType: 'simple'
     }
   );
+
+  // Set up committee typeahead
+  var $committeeFilter = $('#committee-typeahead-filter');
+  var $committeeInput = $('input[name="committee_id"]');
+  $committeeFilter.typeahead({}, typeahead.committeeDataSet);
+  $committeeFilter.on('typeahead:selected', function(event, datum, datasetName) {
+    $committeeInput.val(datum.id);
+  });
+
+  // Set up date picker
+  var $field = $('#file-date');
```

Comment:
Let's make this more general...maybe `#date-picker`? It will be used for more than just filing date.